### DISCUSSION TITLE
How to pin an issue?

### DISCUSSION THREAD

**Discussion by veeableful** [community/community#76380] [New to GitHub] [answered]
Hi, I'm following the instructions [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/pinning-an-issue-to-your-repository) and it says that there is a right sidebar on the Issues page where I can click on __Pin issue__. However, I can't seem to find that sidebar and button in my repository's Issues page. Can someone let me know where I'm going wrong? Thanks!

**Comment by MegatIrfan** [comment]
[
![Screenshot_173](https://github.com/community/community/assets/114043334/5141763c-270d-4ba9-abe0-023facab6e51)
](url)

You can see/click on top Right to see the issue :)

**Comment by veeableful** [comment]
Hi @MegatIrfan, thanks for the response! That seems show issues from every repository but I would like to know if there is a way to pin an issue on a specific repository. I have a screenshot here on my repository's Issues page where it doesn't show a sidebar or pin button.

<img width="1436" alt="Screenshot 2023-11-22 at 11 23 20 AM" src="https://github.com/community/community/assets/1069338/9dd0c3eb-0da6-4007-9179-a13c988a37f4">

**Reply by MegatIrfan** [reply]
![Screenshot_174](https://github.com/community/community/assets/114043334/41e0bde4-0cce-4836-a702-d3df095d2c61)

i think you must tick first then add label or mark as  
https://learntheweb.courses/topics/github-issues/

**Comment by veeableful** [comment]
Hi @MegatIrfan, I can mark issue as open or closed or give label to issues but I don't see an option to pin the issue. Could you show me that option?

**Reply by MegatIrfan** [reply, marked as answer]
![Screenshot_175](https://github.com/community/community/assets/114043334/d7729c6a-1fe1-48a0-b151-6c7612ae1afc)
Open the issue then you can see the right bottom to pin the issue

**Reply by 0843929688** [reply]
OTP

**Reply by 0843929688** [reply]
6102051

**Reply by 0843929688** [reply]
0355919990

**Reply by 0843929688** [reply]
> 6102051

**Comment by veeableful** [comment]
Ahh I see now, thank you! I thought I could pin on the Issues page. I misunderstood the guide and thought that clicking meant selecting the issue on the Issues page with the checkbox.

**Reply by MegatIrfan** [reply]
I'm glad I could help you! :)

**Reply by sohang3112** [reply]
@veeableful I thought the same thing you did - Github should really update the guide to clarify this better.
